### PR TITLE
small clarification

### PR DESCRIPTION
**Pull request by brubakerjeff**
I was looking for the home app on home assistant. I think this change will help the next person.

<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/9812"></a>

### CODE REVIEW

File: source/_components/homekit.markdown [hunk 1]
```diff
@@ -162,7 +162,7 @@ homekit:
 After Home Assistant has started, the entities specified by the filter are exposed to HomeKit if they are [supported](#supported-components). To add them:
 
 1. Open the Home Assistant frontend. A new card will display the `pin code`. Note: If pin code is not displayed, check "Notifications" (the bell icon) in the upper-right of the Home Assistant UI.
-1. Open the `Home` app.
+1. Open the `Home` app on an iOS Device.
```
Comment (frenck) [inline source/_components/homekit.markdown:165]: The `Home` is not exclusively for iOS.

**Comment by frenck** [comment]
Hi there @brubakerjeff,

The change is not completely true, furthermore, the first sentence of this page covers it pretty well:

> The `homekit` integration allows you to forward entities from Home Assistant to Apple HomeKit, so they can be controlled from Apple's Home app and Siri.

Closing this PR, nevertheless, thank you for taking the time to contribute to our project. That is appreciated 👍

**Comment by brubakerjeff** [comment]
Thank you

Just for my education where else would one find the “home” entity
referenced in the PR?

**Comment by frenck** [comment]
It is installed by default on Apple devices. If one removed it, one could get it from the Apple Appstore.

**Comment by brubakerjeff** [comment]
Re:iOS

I see the iOS statement was overconstraining

   -
   - “Apple iPhone, iPad, or iPod touch (*running* *run* iOS 10 or later)
   - Apple Watch (*running* WatchOS 3 or later)
   - Apple TV (fourth generation)
   - Apple HomePod.
   - Macs with macOS 10.14 Mojave or later.”

Yes close
Thanks
Jeff

**Pull request closed without merging**